runtime/runtime_controller.cc

```diff
@@ -20,6 +20,8 @@
 
 namespace flutter {
 
+const uint64_t kFlutterDefaultViewId = 0llu;
```

Comment:
Should this be "implicitView" instead of default view? We are trying to remove the annotation that flutter has a default view. In some modes, it may run with an implicitly created view, though...